src/coreclr/classlibnative/bcltype/arraynative.cpp

```diff
@@ -1154,23 +1154,25 @@ FCIMPL2_IV(void, ArrayNative::InitializeArray, ArrayBase* pArrayRef, FCALLRuntim
 }
 FCIMPLEND
 
-FCIMPL3(void*, ArrayNative::GetSpanDataFrom, FCALLRuntimeFieldHandle structField, FCALLRuntimeTypeHandle targetType, INT32* count)
+FCIMPL3_VVI(void*, ArrayNative::GetSpanDataFrom, FCALLRuntimeFieldHandle structField, FCALLRuntimeTypeHandle targetTypeUnsafe, INT32* count)
```

Comment:
What is the difference between `FCIMPL3_VVI` and `FCIMPL3`?

Reply:
Sigh. The Windows C++ compiler fastcall x86 calling convention is somewhat close to the JIT Windows X86 calling convention, but not identical. In particular, if a function takes structures as arguments, there are differences, which I do not remember the exact details of (see the comments around these defines for more details). Its close enough that it is possible to paper over the differences with macros, and what is done is that the V represents a struct argument and I stands for "normal" argument passed in an argument register. So, this function with the `_VVI` suffix takes 2 structure arguments and a third "normal" argument.

Reply:
...sadness again covers the land...